### PR TITLE
Localize Spanish language

### PR DESCRIPTION
**Pull request by Aboullos**
Continuation from https://github.com/keycloak/keycloak/pull/24467 applying the merge requests

### CODE REVIEW

**Review by jonkoops — DISMISSED** [review]
LGTM code-wise, can't say anything about the translations themselves.

File: themes/src/main/resources-community/theme/base/account/messages/messages_es.properties [hunk 1]
```diff
@@ -146,4 +146,272 @@ invalidPasswordNotUsernameMessage=Contraseña incorrecta: no puede ser igual al
...
+role_manage-account-links=Administrar enlaces de cuenta
+role_manage-consent=Gestionar los consentimientos
+role_uma_authorization=Obtener permisos
+client_account-console=Consola de cuentas
+client_admin-cli=Administrador CLI
+
+device-activity=Actividad del dispositivo
+
+availableRoles=Roles disponibles
+totpStep3DeviceName=Proporcione un nombre de dispositivo para ayudarlo a administrar sus dispositivos OTP.
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_es.properties:226]: ayudarle

File: themes/src/main/resources-community/theme/base/account/messages/messages_es.properties [hunk 2]
```diff
@@ -146,4 +146,272 @@ invalidPasswordNotUsernameMessage=Contraseña incorrecta: no puede ser igual al
...
+authenticatorDefaultStatus=Por defecto
+authenticatorChangePhone=Cambiar el número de teléfono
+
+#Authenticator - Mobile Authenticator setup
+authenticatorMobileSetupTitle=Configuración de autenticador móvil
+smscodeIntroMessage=Ingrese su número de teléfono y se enviará un código de verificación a su teléfono.
+mobileSetupStep1=Instale una aplicación de autenticador en su teléfono. Las aplicaciones enumeradas aquí son compatibles.
+mobileSetupStep2=Abra la aplicación y escanee el código de barras:
+mobileSetupStep3=Ingrese el código único proporcionado por la aplicación y haga clic en Guardar para finalizar la configuración.
+scanBarCode=¿Quieres escanear el código de barras?
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/account/messages/messages_es.properties:350]: ¿Quiere escanear...

File: themes/src/main/resources-community/theme/base/admin/messages/messages_es.properties [hunk 3]
```diff
@@ -6,3 +6,63 @@ invalidPasswordMinSpecialCharsMessage=Contraseña incorrecta: debe contener al m
...
+ldapErrorCantEnableUnsyncedAndImportOff=No puede deshabilitar los usuarios de importación cuando el modo de proveedor LDAP no está sincronizado
+ldapErrorMissingGroupsPathGroup=La ruta de grupos no existe: crea el grupo en la ruta especificada primero
+ldapErrorValidatePasswordPolicyAvailableForWritableOnly=Validar la política de contraseña es aplicable solo con el modo de edición de escritura
+
+clientRedirectURIsFragmentError=Las URIs de redirección no deben contener un fragmento URI
+clientRootURLFragmentError=La URL de la raíz no debe contener un fragmento de URL
+clientRootURLIllegalSchemeError=URL de raíz utiliza un esquema ilegal
+clientBaseURLIllegalSchemeError=URL base utiliza un esquema ilegal
+backchannelLogoutUrlIllegalSchemeError=URL de inicio de sesión de BackChannel utiliza un esquema ilegal
+clientRedirectURIsIllegalSchemeError=Un URI de redirección utiliza un esquema ilegal
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/admin/messages/messages_es.properties:33]: Una URI

File: themes/src/main/resources-community/theme/base/email/messages/messages_es.properties [hunk 4]
```diff
@@ -19,3 +19,33 @@ eventUpdatePasswordBodyHtml=<p>Tu contraseña se ha actualizado el {0} desde {1}
...
+emailUpdateConfirmationBody=Para actualizar su cuenta {2} con la dirección de correo electrónico {1}, haga clic en el enlace a continuación \ n \ n {0} \ n \ n \ nthis el enlace expirará dentro de {3}. \ N \ ni si no desea continuar conEsta modificación, solo ignora este mensaje.
+emailUpdateConfirmationBodyHtml=<P> Para actualizar su cuenta {2} con la dirección de correo electrónico {1}, haga clic en el enlace a continuación </p> <p> <a href="{0HITH> {0} </a> </p> <</p> <P> Este enlace expirará dentro de {3}. </p> <p> Si no desea continuar con esta modificación, simplemente ignore este mensaje. </p>
+emailTestSubject=[KeyCloak] - Mensaje de prueba SMTP
+emailTestBody=Este es un mensaje de prueba
+emailTestBodyHtml=<p> Este es un mensaje de prueba </p>
+identityProviderLinkSubject=Enlace {0}
+identityProviderLinkBody=Alguien quiere vincular su cuenta "{1}" con la cuenta "{0}" del usuario {2}. Si ha sido usted, haga clic en el siguiente enlace para vincular las cuentas \ n \ n {3} \ n \ nEste enlace expirará en {5}. \ n \ nSi no desea vincular la cuenta, simplemente ignore este mensaje. Si vincula las cuentas, podrá iniciar sesión en {1} a través de {0}.
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/email/messages/messages_es.properties:30]: The message is OK, but take care with the spaces in the `\n`.

File: themes/src/main/resources-community/theme/base/email/messages/messages_es.properties [hunk 5]
```diff
@@ -19,3 +19,33 @@ eventUpdatePasswordBodyHtml=<p>Tu contraseña se ha actualizado el {0} desde {1}
...
+# units for link expiration timeout formatting
+# for languages which have more unit plural forms depending on the value (eg. Czech and other Slavic langs) you can override unit text for some other values like described in the Java choice format which is documented here. For Czech, it would be '{0,choice,0#minut|1#minuta|2#minuty|2<minut}'
+# https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/MessageFormat.html
+# https://docs.oracle.com/en/java/javase/17/docs/api/java.base/java/text/ChoiceFormat.html
+linkExpirationFormatter.timePeriodUnit.seconds={0,seleccione,0#segundos|1#segundo|1<segundos}
+linkExpirationFormatter.timePeriodUnit.minutes={0,seleccione,0#minutos|1#minuto|1<minutos}
+linkExpirationFormatter.timePeriodUnit.hours={0,seleccione,0#horas|1#hora|1<horas}
+linkExpirationFormatter.timePeriodUnit.days={0,seleccione,0#días|1#día|1<días}
+
+emailVerificationBodyCode=Verifique su dirección de correo electrónico ingresando en el siguiente código. \ N \ n {0} \ n \ n.
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/email/messages/messages_es.properties:49]: Just the `\n` are not ok.

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 6]
```diff
@@ -252,3 +252,290 @@ access-denied=Acceso denegado
...
+bluetooth=bluetooth
+internal=Interna
+unknown=Desconocida
+
+termsPlainText=Términos y condiciones a definir.
+termsAcceptanceRequired=Debe aceptar nuestros términos y condiciones.
+acceptTerms=Estoy de acuerdo con los términos y condiciones
+
+
+hidePassword=Contraseña oculta
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:293]: Ocultar contraseña

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 7]
```diff
@@ -252,3 +252,290 @@ access-denied=Acceso denegado
...
+delegationCompleteHeader=Inicio de sesión correcto
+delegationCompleteMessage=Puede cerrar esta ventana del navegador y volver a la aplicación de su consola.
+delegationFailedHeader=Error de inicio de sesión
+delegationFailedMessage=Puede cerrar esta ventana del navegador y volver a la aplicación de su consola e intentar iniciar sesión nuevamente.
+
+invalidPasswordMaxLengthMessage=Contraseña no válida: longitud máxima {0}.
+invalidPasswordNotEmailMessage=Contraseña no válida: no debe ser igual al correo electrónico.
+invalidPasswordGenericMessage=Contraseña no válida: la nueva contraseña no coincide con las políticas de contraseña.
+
+successLogout=Estás desconectado
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:400]: Está desconectado

File: themes/src/main/resources-community/theme/base/login/messages/messages_es.properties [hunk 8]
```diff
@@ -252,3 +252,290 @@ access-denied=Acceso denegado
...
+standardFlowDisabledMessage=l cliente no puede iniciar el inicio de sesión del navegador con el tipo de respuesta dado. El flujo estándar está deshabilitado para el cliente.
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/base/login/messages/messages_es.properties:401]: `El cliente`

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_es.properties [hunk 9]
```diff
@@ -1 +1,178 @@
...
+signedOutSession=Desconectado {0}/{1}
+lastAccessedOn=Último accedido
+clients=Clientes
+started=Comenzó
+expires=Expira
+ipAddress=dirección IP
+
+# Resources page
+resourceName=Nombre del recurso
+nextPage=Próxima
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_es.properties:53]: Siguiente

File: themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_es.properties [hunk 10]
```diff
@@ -1 +1,178 @@
...
+webauthn-passwordless-display-name=Clave de seguridad
+webauthn-passwordless-help-text=Use su clave de seguridad para iniciar sesión sin contraseña.
+basic-authentication=Autenticación básica
+invalidRequestMessage=Solicitud no válida
+
+# Applications page
+applicationsPageTitle=Aplicaciones
+applicationsPageSubTitle=Administre sus permisos de solicitud
+internalApp=Interna
+thirdPartyApp=Tercero
```
Comment (rmartinc) [inline themes/src/main/resources-community/theme/keycloak.v2/account/messages/messages_es.properties:124]: Aplicación de terceros

**Review by rmartinc — CHANGES_REQUESTED** [review]
Thanks a lot for teh effort @Aboullos!
Just a few more and it's done... :smile:

**Comment by jonkoops** [comment]
@rmartinc Just FYI, you can also [provide suggestions](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/reviewing-changes-in-pull-requests/commenting-on-a-pull-request#adding-comments-to-a-pull-request) in GitHub that can be committed directly from the comments.

**Comment by Aboullos** [comment]
@rmartinc thanks for the review, change requests applied

**Review by rmartinc — APPROVED** [review]
Thanks a lot @Aboullos! Approved!

**Review by jonkoops — APPROVED** [review]

**Review by ssilvert — APPROVED** [review]

**Pull request merged**